pkg/gadgets/top/ebpf/piditer/iter.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
```

Comment:
This code could be useful for other gadgets so maybe we can reorganize this in another way and move it out from this folder. Anyway, it will be done once at least another gadget starts using it.

Reply:
Yeah, I was thinking kind of the same, however I didn't really know where to put it. For all other gadgets, the ebpf part holds the main functionality. In ebpftop it's only a helper that is used to get the processes that are using the ebpf programs. Then, however, we would have ebpf code in other packages than pkg/gadgets (and generate in there), so I'm not sure what to do. But maybe it's something we can discuss in another PR?

Reply:
Absolutely, it will be manage in another PR. I just wanted to mention it.